### ISSUE TITLE
Draconic Evolution - Angel Ring consumed but no flight at basic tier

### ISSUE THREAD

**Issue by Refisio** [AllTheMods/ATM-3#562]
## Expected Behavior

Upon crafting Draconic Evolution's Wyvern Chestpiece, the Angel Ring should be consumed and the user should be able to fly with Draconic Evolution's Wyvern Chestpiece.

## Current Behavior

Upon crafting Draconic Evolution's Wyvern Chestpiece, the Angel Ring is consumed and the user is *not* able to fly with Draconic Evolution's Wyvern Chestpiece.

## Possible Solution

As suggested in the All the Mods 3 Discord, the Wyvern Chestplate should be changed so that a Diamond Chestplate replaces the Angel Ring in the center, and the Draconic Chestplate recipe, found in the Fusion Crafter, should be changed to incorporate the Angel Ring.

## Steps to Reproduce

1. Craft Draconic Evolution's Wyvern Chestplate using the Angel Ring from Angel Ring to Baubles.
2. Open Draconic Evolution's HUD Menu.
3. Select the Wyvern Chestplate. Find that there is no options for said Chestplate.
4. Attempt to fly using the Wyvern Chestplate by double-tapping the Jump key.

**Comment by OrdinatorStouff** [comment]
I can confirm that when I made this recipe I thought the basic Draconic armor could fly (but I was WRONG)

**Comment by kjmaster1** [comment]
Changing a Fusion Crafter recipe can only be done using the Draconic Evolution JSON file method. Changing the Draconic Chestplate Fusion Crafter recipe would cause the change to occur for both pack modes. Perhaps only the Wyvern Chestplate recipe should be changed to remove the Angel Ring and the Draconic Chestplate recipe remains the same?

**Comment by Refisio** [comment]
While that does sound realistic and fair, what, then, stops me from "donating" the Angel Ring to someone else?

**Comment by Sakata-MC** [comment]
@Refisio You stop yourself from 'donating' the ring to people.  But that aside, hopefully this recipe will be addressed in next update.

**Comment by Yoosk** [comment]
Or we could make it use angel ring as is, but does not consume it on crafting armor.

**Comment by iopredman** [comment]
I just reached this point in my play-through, and was minorly peeved to find that the flight ring is not usable in the wyvern armor recipe, meaning that I had to craft another angel ring in order to build the wyvern armor. If this was the intention, then so be it, but I suggest either adding a recipe for the wyvern armor to include the flight ring instead of the angel ring, or add an uncrafting table (twilight forest) recipe to revert the flight ring to a regular angel ring. 

In other words, the current state forces the player to either make the decision of crafting two angel rings for the wyvern armor or just not make the flight ring, which seems antithetical to the goal of the pack.

**Comment by martmists** [comment]
`On a related note, because of the biome gen, finding a jungle in which to capture parrots for the ring is basically impossible` 
Half of my biomes are jungle sadly, so I wouldn't do anything to change this.
If anything, it should be made so the Draconic Chestplate recipe gets altered to use the angel ring, not the Wyvern Chestplate

**Comment by FallenEagle1** [comment]
Recipes have been altered for 6.0. Angel ring is used in the draconic chestplate instead of the wyvern
